src/core/coerce.c

```diff
@@ -27,7 +27,7 @@ static void boolify_return(MVMThreadContext *tc, void *sr_data);
 static void flip_return(MVMThreadContext *tc, void *sr_data);
 void MVM_coerce_istrue(MVMThreadContext *tc, MVMObject *obj, MVMRegister *res_reg,
         MVMuint8 *true_addr, MVMuint8 *false_addr, MVMuint8 flip) {
-    MVMint64 result;
```

Comment:
`result` should be set somewhere in the `MVM_BOOL_MODE_CALL_METHOD` case in the following switch statement. Or the `if` statement should be modified ( it doesn't make sense to set it to zero twice )